src/get-started/new-typography-scale/index.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: layout-pane.njk
+title: Updating your service to use the new typography scale
```

Comment:
I've called this 'type scale' on the other page – it's the more commonly used phrase.

I think 'type scale' is actually shortened from 'typographic scale'. So it should probably be 'type scale' or 'typographic scale'. I personally prefer the shorter one!

Reply:
type scale sounds like a clear way to describe it.

Reply:
Ok let's use type scale throughout IMO

Reply:
```suggestion
title: Updating your service to use the new type scale
```